.github/pull_request_template.md

```diff
@@ -6,4 +6,5 @@
 
 #### Checklist
 - [ ] Included tests
+- [ ] Edited documentation
```

Comment:
Maybe it would be good to mention where the documentation is. So what do you think about the following?
```suggestion
- [ ] Updated documentation in [README](https://github.com/mojira/arisa-kt/blob/master/README.md) and [wiki](https://github.com/mojira/arisa-kt/wiki)
```